Polyfills/Console/Source/Console.cpp

```diff
@@ -53,15 +53,16 @@ namespace
     void AddMethod(Napi::Object& console, const char* functionName, Babylon::Polyfills::Console::LogLevel logLevel, Babylon::Polyfills::Console::CallbackT callback)
     {
         auto existingFunction = std::make_shared<Napi::FunctionReference>(Napi::Persistent(console.Get(functionName).As<Napi::Function>()));
-        console.Set(functionName, Napi::Function::New(console.Env(), [callback, existingFunction = std::move(existingFunction), logLevel](const Napi::CallbackInfo& info)
-        {
-            InvokeCallback(callback, info, logLevel);
+        console.Set(functionName, Napi::Function::New(
+                                      console.Env(), [callback, existingFunction = std::move(existingFunction), logLevel](const Napi::CallbackInfo& info) {
+                                          InvokeCallback(callback, info, logLevel);
 
-            if (!existingFunction->Value().IsUndefined())
-            {
-                Call(existingFunction->Value(), info);
-            }
-        }, functionName));
+                                          if (!existingFunction->Value().IsUndefined())
+                                          {
+                                              Call(existingFunction->Value(), info);
+                                          }
+                                      },
+                                      functionName));
```

Comment:
ditto

Reply:
This can be solved by moving the lambda declaration to a new line. Lambda body will always align with the declaration.